### PR TITLE
Ppa 549

### PR DESCRIPTION
**Pull request by htmgarcia**
#549

When Blocks is not activated, this banner is displayed:
Screen Shot 2021-11-22 at 16 37 49

@stevejburge the search word we use here is also the same as Permissions? "publishpress-ppcore-install"
Screen Shot 2021-11-22 at 16 39 17
Also banner text may need a review.

### CODE REVIEW

**Comment by stevejburge** [comment]
@htmgarcia Looks good. We can use the label, "publishpress-advg-install"

Are there any more "Authors" integrations we can mention in Blocks?

**Comment by htmgarcia** [comment]
@stevejburge as I can remember displaying multiple authors, and filtering by authors created through PP Authors are the 2 features integrated in Content Display block.

**Comment by htmgarcia** [comment]
@stevejburge Screen Shot 2021-11-23 at 11 46 53

**Comment by stevejburge** [comment]
@htmgarcia Thanks

Title: Showcase your Authors with PublishPress Blocks

- PublishPress Blocks is a free plugin with full support for PublishPress Authors.
- Install this plugin to showcase content by your Authors.
- Use the Content Display block to show your posts in many beautiful layouts.
- PublishPress Blocks has over 20 extra Gutenberg blocks including accordions, galleries, tables, and more.

"Click here to install PublishPress Block"

File: src/modules/multiple-authors/multiple-authors.php [hunk 1]
```diff
@@ -436,6 +437,20 @@ public function print_configure_view()
...
 
+        /**
+         * Check if Blocks Free or Pro are active
+         */
+        public function pp_blocks_is_active()
+        {
+            if (
+                !MultipleAuthors\Classes\Utils::isPluginActive('advanced-gutenberg.php')
+                && !MultipleAuthors\Classes\Utils::isPluginActive('advanced-gutenberg-pro.php')) {
+            	define('PP_AUTHORS_BLOCKS_INSTALLED', false);
```
Comment (andergmartins) [inline src/modules/multiple-authors/multiple-authors.php:448]: @htmgarcia I think we should only define the constant here if is not defined. If anyone runs this plugin twice (by mistake or for any reason...) we will break the admin site with a PHP warning about the constant is already defined.

I suggest adding the conditional if constant is defined, at the beginning of the function. And define it once, based on the bool result of the expression you have on this IF right now.

**Review by andergmartins — CHANGES_REQUESTED** [review]

File: src/modules/multiple-authors/multiple-authors.php (hunk 1 above)
Comment (ojopaul) [reply src/modules/multiple-authors/multiple-authors.php:448]: This issue is fixed now

Comment (andergmartins) [reply src/modules/multiple-authors/multiple-authors.php:448]: Thanks @ojopaul I will merge this one and then will merge your changes as well.

**Review by andergmartins — APPROVED** [review]

**Pull request merged**